variables.tf

```diff
@@ -173,3 +173,25 @@ variable "kuberhealthy" {
   default     = false
 }
 
+// GCP Artifact
+variable "artifact_location" {
+  description = "artifact registry repository Location"
+  type        = string
+  default     = "us-central1" 
+}
+variable "artifact_repository_id" {
+  description = "artifact registry repository Name, Defaul Cluster Name"
+  type        = string
+  default     = "oci"
+}
+variable "artifact_description" {
+  description = "artifact registry repository Description"
```

Comment:
Fix weird casing

Reply:
Suggestion?

Reply:
Never mind. It should be fixed in the other descriptions and in terraform-google-jx as well. Some other time...